apps/arena/lib/arena/game_updater.ex

```diff
@@ -353,11 +351,17 @@ defmodule Arena.GameUpdater do
 
     broadcast_player_dead(state.game_state.game_id, victim_id)
 
-    GameTracker.push_event(
-      self(),
-      {:kill, %{id: killer.id, character_name: killer.aditional_info.character_name},
-       %{id: victim.id, character_name: victim.aditional_info.character_name}}
-    )
+    case Map.get(game_state.players, killer_id) do
```

Comment:
This is the fix for the killing zone bug, right?

Reply:
Yes, this commit specifically [a6c7000](https://github.com/lambdaclass/mirra_backend/pull/609/commits/a6c70007dca97f389428662604e6513cb481ee79)